registry/schemas/xregistry_openapi_messagedefinition.json

```diff
@@ -0,0 +1,960 @@
+{
+  "openapi": "3.0.6",
+  "info": {
+    "title": "xRegistry API",
```

Comment:
same comments from xregistry_openapi_endpoints apply here (make it specific to message defs)